cmd/edit/ingress/cmd.go

```diff
@@ -205,6 +226,10 @@ func run(cmd *cobra.Command, argv []string) {
 	}
 
 	r.Reporter.Debugf("Updating ingress '%s' on cluster '%s'", ingress.ID(), clusterKey)
+	if private == nil || len(routeSelectors) == 0 {
```

Comment:
Not sure how to set this up for any given moment
Thought of doing a function that would take DTO like object as input, but we would need to change that too if we ever decide for more conditions

Open to better suggestions

Reply:
L228 should be inside that `if` block